source/agora/consensus/PreImage.d

```diff
@@ -343,15 +343,18 @@ public struct PreImageCycle
         is part of consensus.
 
         Params:
-          secret = The secret key of the node, used as part of the hash
-                   to generate the cycle seeds
+            secret = The secret key of the node, used as part of the hash
+                to generate the cycle seeds
+            calc_index = The flag to determine to calculating the cycle index
+                or just getting a random seed, or the first pre-image
```

Comment:
I'm a little bit confused by this sentence because it's saying it will do one of **three** things but it's a boolean.

Reply:
> I'm a little bit confused by this sentence because it's saying it will do one of **three** things but it's a boolean.

The description should be changed. Thanks.